1.6/Defs/Incidents/MSSFP_Incidents.xml

```diff
@@ -27,13 +26,12 @@
         <usableForArt>true</usableForArt>
         <rulePack>
             <rulesStrings>
-                <li>tale_noun->[LOVERA_nameDef] whisked [LOVERB_nameDef] away for a romantic retreat</li>
```

Comment:
I think your autoformatter is being overzealous. Can you change them back to `->`?